.github/CODEOWNERS

```diff
@@ -4,5 +4,7 @@
 package.json                         @danjm @whymarrh @Gudahtt
 yarn.lock                            @danjm @whymarrh @Gudahtt
 ui/                                  @danjm @whymarrh @Gudahtt
-app/scripts/controllers/transactions @frankiebee @whymarrh
+app/scripts/controllers/transactions @frankiebee @whymarrh @jennypollack
 .circleci/scripts/deps-install.sh    @kumavis @Gudahtt
+app/scripts/controllers/network      @jennypollack @rekmarks
+app/scripts/controllers/permissions  @rekmarks
```

Comment:
Could you think of someone else that could share this responsibility? Maybe @danjm ? Having a single code-owner should generally be avoided, so that we aren't depending entirely on one person for review.

Reply:
I was thinking about that, and I wasn't sure who to pick, but I think @danjm would be best